### ISSUE TITLE
No 1D line segment support

### ISSUE THREAD

**Issue by Cloudmersive** [mathnet/mathnet-spatial#171]
There are 2D and 3D line segments but no 1D line segments

**Comment by JohanLarsson** [comment]
What is an 1D line segment? A range?

**Comment by Cloudmersive** [comment]
Yeah, basically.  Here is an example: https://eli.thegreenplace.net/2008/08/15/intersection-of-1d-segments

**Comment by jkalias** [comment]
What would be your case then?
